exporter/templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.17.xml

```diff
@@ -0,0 +1,30 @@
+<entry>
+  <observation classCode="OBS" moodCode="RQO" {{negationIndicator .EntrySection.Entry}}>
+    <!-- Consolidated Plan of Care Activity Observation
+        templateId (Implied Template) -->
+    <templateId root="2.16.840.1.113883.10.20.22.4.44" extension="2014-06-09"/>
+    <!-- Diagnostic Study, Order template -->
+    <templateId root="2.16.840.1.113883.10.20.24.3.17" extension="2014-12-01"/>
+    <id root="1.3.6.1.4.1.115" extension="{{.EntrySection.Entry.ID}}"/>
```

Comment:
In some templates we're using `newRandom` in the extension, in some the ID. We should pick a standard and make sure all of them comply with it. It seems like most of the JSON `Entry`s don't have a `CdaIdentifier`, so this field might be blank, which is why we were using `newRandom`